components/gitpod-protocol/src/gitpod-service.ts

```diff
@@ -294,6 +294,7 @@ export interface GitpodServer extends JsonRpcServer<GitpodClient>, AdminServer,
     createOrUpdateStripeCustomerForTeam(teamId: string, currency: string): Promise<void>;
     createOrUpdateStripeCustomerForUser(currency: string): Promise<void>;
     subscribeTeamToStripe(teamId: string, setupIntentId: string): Promise<void>;
+    getStripePortalUrl(attributionId: string): Promise<string>;
```

Comment:
Why make this name ambiguous?

```suggestion
    getStripePortalUrlForUser(attributionId: string): Promise<string>;
```

Reply:
Ah sorry, just saw the `attributionId`